modules/doc/content/newsletter/2023/2023_03.md

```diff
@@ -12,3 +12,8 @@ for a complete description of all MOOSE changes.
 ## PETSc-level Changes
 
 ## Bug Fixes and Minor Enhancements
+
+### VectorPostprocessor Parallel Bugfix
+
+A [parallel bug identified by ANL researchers](https://github.com/idaholab/moose/issues/23514),
+was fixed by [defaulting to asking for parallel synchronized VPP values](https://github.com/idaholab/moose/pull/23588).
```

Comment:
```suggestion
- A [VectorPostprocessor parallel bug identified by ANL researchers](https://github.com/idaholab/moose/issues/23514), was fixed by [defaulting to asking for parallel synchronized VPP values](https://github.com/idaholab/moose/pull/23588).
```

Thank you for the news entry! This section should be bulleted, since they are intended for single sentence mentions of fixes. Anything that requires longer form sentences should go in the MOOSE section above.